Upstream: https://trac.gajim.org/ticket/7324

exporting patch:
# HG changeset patch
# User Nikolay Orlyuk <virkony@gmail.com>
# Date 1363941015 -7200
# Node ID 8ad5efe338b6d61fbfdcee10e2d9b811d4c4c89f
# Parent  17df4ebe8da82f26e0b1dc0745b278c4c6b30dc6
update POTFILES.skip generation

- rewrite entries generation for plugins/
- make generation more friendly for updates
- add src/command_system/mapping.py

diff -r 17df4ebe8da8 -r 8ad5efe338b6 autogen.sh
--- a/autogen.sh	Tue Mar 19 13:32:03 2013 +0100
+++ b/autogen.sh	Fri Mar 22 10:30:15 2013 +0200
@@ -16,11 +16,15 @@
   po/POTFILES.in; done \
   && ls -1 data/gajim.desktop.in.in \
   src/*.py src/common/*.py src/command_system/implementation/*.py src/common/zeroconf/*.py src/plugins/*.py | grep -v ipython_view.py >> \
-  po/POTFILES.in \
-  && echo -e "data/gajim.desktop.in\nsrc/ipython_view.py" > po/POTFILES.skip  || exit 1
-  if [ $(find plugins/ -name '*.py' | wc -l) -gt 0 ];then
-    ls -1 plugins/*/*.py plugins/*/*.ui >> po/POTFILES.skip
-  fi
+  po/POTFILES.in || exit 1
+  for file in \
+      data/gajim.desktop.in \
+      src/ipython_view.py \
+      src/command_system/mapping.py
+  do
+      echo "$file"
+  done > po/POTFILES.skip || exit 1
+  find plugins/ -name '*.py' -o -name '*.ui' >> po/POTFILES.skip
   if test -z `which pkg-config 2>/dev/null`;then
     echo "***Error: pkg-config not found***"
 	echo "See README.html for build requirements."
